src/const.js

```diff
@@ -20,14 +20,15 @@ const TEXTS_FOR_EMPTY_SHEET = {
   emptyPointsListForFuture: 'There are no future events now',
 };
 
-const sortList = {
-  day: '',
-  event: 'disabled',
-  time: '',
-  price: 'checked',
-  offers: 'disabled',
+const SortType = {
+  DAY: 'day',
+  EVENT: 'disabled',
```

Comment:
Предлагаю заменить на 
```suggestion
  EVENT: {
    value: 'event',
    state: 'disabled'
  }
```
для каждого ключа. Кажется что так будет нагляднее